docs/reference.md

```diff
@@ -420,6 +421,11 @@ OpenWhisk has a few system limits, including how much memory an action uses and
 * A user can change the limit when creating the action.
 * A container cannot have more memory allocated than the limit.
 
+### Per action logs (MB) (Default: 10MB)
+* The log limit M is in the range from [0MB..10MB] and is set per action in MB.
```

Comment:
using M here, N above. suggest this:
 +* The log limit N is in the range [0MB..10MB] and is set per action.